ArcGISRuntimeSDKQt_CppSamples/Analysis/FindServiceAreasForMultipleFacilities/FindServiceAreasForMultipleFacilities.qml

```diff
@@ -0,0 +1,58 @@
+// [WriteFile Name=FindServiceAreasForMultipleFacilities, Category=Analysis]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.11
+import Esri.Samples 1.0
+
+Item {
+
+    // add a mapView component
+    MapView {
+        id: view
+        anchors.fill: parent
+
+        RowLayout {
+            anchors {
+                left: parent.left
+                top: parent.top
+            }
+
+            Button {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                text: "Find Service Areas"
+                onClicked: {
+                    model.findServiceAreas();
+                    enabled = false;
```

Comment:
Instead of `enabled = false` I would recommend the property
`enabled: model.taskRunning`

Reply:
Based on the other platforms' implementations of this sample, it seems the button should be permanently disabled once pressed